keyboards/zygomorph/keymaps/default_oled/rules.mk

```diff
@@ -0,0 +1,38 @@
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
+#
+
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE = no        # Mouse keys(+4700)
+EXTRAKEY_ENABLE = yes       # Audio control and System control(+450)
+CONSOLE_ENABLE = no         # Console for debug(+400)
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = no            # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+RGBLIGHT_ENABLE = yes       # Enable global lighting effects. Do not enable with RGB Matrix
+RGBLIGHT_ANIMATIONS = yes   # LED animations
+RGBLIGHT_SPLIT_ENABLE = no  # Split RGBLight Support
+RGB_MATRIX_ENABLE = no      # Enable per-key coordinate based RGB effects. Do not enable with RGBlight (+8500)
+RGB_MATRIX_KEYPRESSES = no  # Enable reactive per-key effects. Can be very laggy (+1500)
+RGBLIGHT_FULL_POWER = yes   # Allow maximum RGB brightness. Otherwise, limited to a safe level for a normal USB-A port
+UNICODE_ENABLE = no         # Unicode
+SWAP_HANDS_ENABLE = no      # Enable one-hand typing
+ENCODER_ENABLE = yes        # Enable rotary encoder (+90)
+OLED_DRIVER_ENABLE = yes    # Enable the OLED Driver (+5000)
+IOS_DEVICE_ENABLE = no      # Limit max brightness to connect to IOS device (iPad,iPhone)
+
+# Do not edit past here
+
+ifeq ($(strip $(IOS_DEVICE_ENABLE)), yes)
```

Comment:
Given this logic is for the config.h, it would make sense for it to be relocated to the keyboards root rules.mk

Reply:
It doesn't work if you relocate it to a different make file due to the order that the make files apply. The rules.mk files are executed in this order: keyboard > revision > keymap > common_features.mk. So for custom features like this one or the others, it needs to be at the lowest level so it properly knows the final value before doing it's work. What would be handy is a keyboard_features.mk file that is executed after keymap but before common_features.mk to handle things like this. However that is not something for this pr.